CalibCalorimetry/HcalPlugins/python/Hcal_Conditions_forGlobalTag_cff.py

```diff
@@ -35,7 +35,7 @@
         qieSlope      = cms.vdouble(0.912,0.917,0.922,0.923),
         mcShape       = cms.int32(125),
         recoShape     = cms.int32(105),
-        photoelectronsToAnalog = cms.double(0.0),
+        photoelectronsToAnalog = cms.double(0.3305),
         darkCurrent   = cms.vdouble(0.0),
```

Comment:
The rest of the code changes looks like really just related to the additional enum, but what is the rationale behind this configuration parameter change?
Why does this value needs to be changed?